src/index.html

```diff
@@ -90,41 +85,39 @@ <h2><a class="anchor" id="regulations"></a>Regulations</h2>
         <li>
           Teams must conduct themselves within the principles of good sporting conduct.
         <li>
-          Any assistance from Smallpeice Trust staff and volunteers is provided without guarantees.
+          Robots must be fully autonomous. No remote control systems or receivers are permitted.
         <li>
-          No remote control systems or radio transmitters or receivers are permitted.
+          Robots must pass a safety inspection before they can compete.
+          <ol>
+            <li>
+              Robots must not have any sharp edges or points.
+            <li>
+              Robots must have all electronics securely mounted and easily removable.
```

Comment:
```suggestion
              Robots must have all electronics securely mounted.
```

The easily removable bit we could do with just assigned mentors keeping an eye in the SP context.